src/common/exceptions/http/badRequestError.ts

```diff
@@ -0,0 +1,14 @@
+import HttpStatus from 'http-status-codes';
```

Comment:
have you talked with @cl-shaharshavit-rnd ? is it the same code?
open an issue about creating a package of common error types

Reply:
yes, i did talk with him